pidtree_bcc/main.py

```diff
@@ -125,6 +127,7 @@ def parse_args():
     parser.add_argument("-c", "--config", type=str, help="yaml file containing subnet safelist information")
     parser.add_argument("-p", "--print-and-quit", action='store_true', default=False, help="don't run, just print the eBPF program to be compiled and quit")
     parser.add_argument("-f", "--output_file", type=str, default='-', help="File to output to (default is STDOUT, denoted by -)")
+    parser.add_argument("-v", "--version", action='store_true', default=False, help="Print the version of pidtree-bcc")
```

Comment:
argparse supports a special action for version info, e.g. `parser.add_argument('-v', '--version', action='version', version='%(prog)s 2.0')`. It will also handle exiting automagically.

Reply:
++